client-app/pages/account/addresses.vue

```diff
@@ -14,13 +17,35 @@
           <div class="flex justify-between items-center mx-5 md:mx-0">
             <h2 class="text-gray-800 text-3xl font-bold uppercase">{{ title }}</h2>
 
-            <VcButton v-if="!editingMode" class="px-3 uppercase" size="sm" is-outline @click="openEditMode()">
+            <VcButton
+              v-if="!editingMode && paginatedAddresses.length"
+              class="px-3 uppercase"
+              size="sm"
+              is-outline
+              @click="openEditMode()"
+            >
               <span class="sm:hidden">{{ $t("pages.account.addresses.add_new_address_button_mobile") }}</span>
               <span class="hidden sm:inline">{{ $t("pages.account.addresses.add_new_address_button") }}</span>
             </VcButton>
           </div>
 
-          <div class="flex flex-col bg-white shadow-sm md:rounded md:border">
+          <!-- Empty view -->
+          <VcEmptyView
+            :text="$t('pages.account.addresses.no_addresses_message')"
+            v-if="!paginatedAddresses.length && !editingMode"
+          >
+            <template #icon>
+              <VcImage :src="'/static/images/account/icons/no-addresses.svg'" />
```

Comment:
Use `src="/static/images/account/icons/no-addresses.svg"` instead - you shouldn't pass simple string like variable.